testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxBoundedContext.java

```diff
@@ -283,6 +286,71 @@ public T receivesEvents(Message firstEvent, Message secondEvent, Message... othe
         return this.receivesEvents(asList(firstEvent, secondEvent, otherEvents));
     }
 
+    /**
+     * Sends off a provided event to the Bounded Context as event from an external source.
+     *
+     *
+     * @param sourceContext
+     *         a name of the Bounded Context external events come from
+     * @param messageOrEvent
+     *         an event message or {@link io.spine.core.Event}. If an instance of {@code Event} is
+     *         passed, it will be posted to {@link io.spine.server.integration.IntegrationBus}
+     *         as is.
+     *         Otherwise, an instance of {@code Event} will be generated basing on the passed
+     *         event message and posted to the bus.
+     * @return current instance
+     * @apiNote Returned value can be ignored when this method invoked for test setup
+     */
+    @CanIgnoreReturnValue
+    public T receivesExternalEvent(BoundedContextName sourceContext, Message messageOrEvent) {
+        return this.receivesExternalEvents(sourceContext, singletonList(messageOrEvent));
+    }
+
+    /**
+     * Sends off provided events to the Bounded Context as events from an external source.
+     *
+     * <p>The method accepts event messages or instances of {@link io.spine.core.Event}.
+     * If an instance of {@code Event} is passed, it will be posted to
+     * {@link io.spine.server.integration.IntegrationBus} as is.
+     * Otherwise, an instance of {@code Event} will be generated basing on the passed event
+     * message and posted to the bus.
+     *
+     * @param sourceContext
+     *         a name of the Bounded Context external events come from
+     * @param firstEvent
+     *         an external event to be dispatched to the Bounded Context first
+     * @param secondEvent
+     *         an external event to be dispatched to the Bounded Context second
+     * @param otherEvents
+     *         optional external events to be dispatched to the Bounded Context
+     *         in supplied order
+     * @return current instance
+     * @apiNote Returned value can be ignored when this method invoked for test setup
```

Comment:
We have agreed to put periods in `@apiNote` and `@implNote`, as these are sentences.